doc/client_lua_api.md

```diff
@@ -702,10 +702,8 @@ Call these functions only at load time!
 ### Map
 * `minetest.get_node(pos)`
     * Returns the node at the given position as table in the format
-      `{name="node_name", param1=0, param2=0}`, returns `{name="ignore", param1=0, param2=0}`
-      for unloaded areas.
-* `minetest.get_node_or_nil(pos)`
-    * Same as `get_node` but returns `nil` for unloaded areas.
+      `{name="node_name", param1=0, param2=0}`, returns `nil`
+      for unloaded areas or flavour limited areas.
```

Comment:
either make it return ignore, or rename the function to `get_node_or_nil` please (in order to make it consistent.